plugins/base/src/main/kotlin/transformers/pages/tags/SinceKotlinTagContentProvider.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.dokka.base.transformers.pages.tags
+
+import org.jetbrains.dokka.DokkaConfiguration
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder.DocumentableContentBuilder
+import org.jetbrains.dokka.model.doc.CustomTagWrapper
+import org.jetbrains.dokka.pages.ContentKind
+import org.jetbrains.dokka.pages.TextStyle
+
+object SinceKotlinTagContentProvider : CustomTagContentProvider {
+
+    private val TAG_NAME = "Since Kotlin"
+
+    override fun DocumentableContentBuilder.contentForDescription(
+        sourceSet: DokkaConfiguration.DokkaSourceSet,
+        customTag: CustomTagWrapper
+    ) {
+        sinceKotlinBlock(sourceSet, customTag)
+    }
+
+    override fun DocumentableContentBuilder.contentForBrief(
+        sourceSet: DokkaConfiguration.DokkaSourceSet,
+        customTag: CustomTagWrapper
+    ) {
+        sinceKotlinBlock(sourceSet, customTag)
```

Comment:
I would make it as single line here. (Anton suggested this).
`text(tag.name + " ", styles = setOf(TextStyle.Bold))`